bin/grommet.js

```diff
@@ -43,8 +43,8 @@ var tasks = {
         process.chdir('./' + app);
 
         var templateFolder = path.join(grommetPath, 'templates/init/**');
-        var mobileIcon = path.join(grommetPath, 'mobile-app-icon.png');
-        var shortcutIcon = path.join(grommetPath, 'shortcut-icon.png');
+        var mobileIcon = path.join(grommetPath, 'src/img/mobile-app-icon.png');
+        var shortcutIcon = path.join(grommetPath, 'src/img/shortcut-icon.png');
 
         gulp.src(mobileIcon).pipe(gulp.dest('./src/img'));
```

Comment:
I believe here is the destination folder you can see we use `src/img` here.

But the place where we grab the image from should be `path.join(grommetPath, 'img/shortcut-icon.png');`